teuthology/config.py

```diff
@@ -209,6 +210,9 @@ def get_ceph_qa_suite_git_url(self):
     def get_ceph_git_url(self):
         return (self.ceph_git_url or
                 self.ceph_git_base_url + 'ceph-ci.git')
+    def get_teuthology_git_url(self):
+        return self.teuthology_git_url
```

Comment:
this might be the same as the other similar methods, like:
```
        return (self.teuthology_git_url or
                 self.ceph_git_base_url + 'teuthology.git')
```